src/api/parse/test.ts

```diff
@@ -4,7 +4,7 @@ import { parseEmail } from "./main";
 const PATH = "configs/test_email.json";
 
 async function test() {
-  let apiKey = process.env.OPENAI_API_KEY || "";
+  let apiKey = process.env.openaiApiKey || "";
```

Comment:
The `OPENAI_API_KEY` is a environment variable, which does not need to follow JS naming conventions. Also, the convention for openai api key is to store it in the environment variable `OPENAI_API_KEY`.